spec/decision_service_spec.rb

```diff
@@ -109,6 +114,7 @@
     end
 
     it 'should respect forced variations within mutually exclusive grouped experiments' do
+      expect(config.set_forced_variation('group1_exp2', 'forced_group_user1', 'g1_e2_v2')).to eq(TRUE)
```

Comment:
@faisal-iqbal  I will get back to you, it's whitelisted as you mentioned and setforced variation not supposed to be here. It is getting variation from the config file.

Reply:
Removed set_forced_variation from here